client/state/products-list/test/selectors.js

```diff
@@ -33,6 +33,9 @@ jest.mock( 'state/sites/plans/selectors', () => ( {
 jest.mock( 'lib/plans', () => ( {
 	applyTestFiltersToPlansList: jest.fn( x => x ),
 	getPlan: jest.fn(),
+	getTermDuration: jest.fn(
+		term => ( term === 'TERM_MONTHLY' ? 31 : term === 'TERM_ANNUALLY' ? 365 : 730 ) // eslint-disable-line no-nested-ternary
+	),
```

Comment:
why do we have to mock this?

Reply:
@dmsnell we were mocking it because this function had to be present in order for selectors.js to work jest.mock mocks entire module and not just requested properties. But not mocking it is a better way.